ts/utils/clipboard.ts

```diff
@@ -0,0 +1,20 @@
+import { Toast } from "native-base";
+import { Clipboard } from "react-native";
+
+import I18n from "../i18n";
+
+export const copyToClipboard = (
```

Comment:
Please add a comment that explain the purpose of this

Reply:
perhaps we could also rename it `copyToClipboardWithToast` to make it clear what it does

Reply:
I think Toast is an internal implementation detail.

Reply:
agreed, `copyToClipboardWithFeedback` ?

Reply:
As you suggested clipboardSetStringWithFeedback